src/input.c

```diff
@@ -543,6 +543,12 @@ static void input_scroll_continue(rct_window *w, int widgetIndex, int state, int
 	int scroll_part, scroll_id;
 	int x2, y2;
 
+	if (w == NULL)
+	{
+		log_error("Null window received");
```

Comment:
I'd favour an assertion here... we want to be aware when this situation does occur and prevent it.